packages/apps/job-launcher/server/src/modules/job/job.controller.ts

```diff
@@ -81,4 +83,14 @@ export class JobController {
   public async cancelCronJob(): Promise<any> {
     return this.jobService.cancelCronJob();
   }
+
+  @Public()
+  @UseGuards(SignatureAuthGuard)
+  @Post('/cvat/escrow-failed-webhook')
```

Comment:
Perhaps it can wok with FORTUNE type as well, but fortune exchange oracle doesn't have this implementation for now. I can simplify this route for the usage further

Reply:
Why do you need the oracleType? You can read the job type from database based on the escrow address

Reply:
I can, but there several extra step:
1. Get job entity from DB using escrow address

Reply:
I can, but there several extra step:
1. Get job entity from DB using `escrowAddress`
2. Get manifest my `manifestUrl` and retrieve from it `requestType`

Also, server need to do it any time when receive request, before it will check signature, If someone starts sending requests very often, the load on the database will increase.

Reply:
Actually why do you even need to know the job type?

Reply:
We need it to
1. Be sure which address of exchange oracle need to be a signer and compare it with data got it verify signature method 
2. If we will check both addresses, an one of this will be correct, it can means that wrong oracle have called wrong endpoint and pass verification. For example when fortune ex oracle will call /CVAT endpoint and failed the job